md/configurable-archive.md

```diff
@@ -15,17 +15,20 @@ In general we recommended that you set all attributes to **true** or all to **fa
 
 * To archive the ProcessInstance comments when the process instance completes, set `org.bonitasoft.engine.core.process.comment.model.SComment` to true. Comments entered in a task during execution are archived in executed tasks in archived cases.
 * To archive a process instance each time its state changes, set `org.bonitasoft.engine.core.process.instance.model.SProcessInstance` to true.
-* To archive transition instances, set `org.bonitasoft.engine.core.process.instance.model.STransitionInstance` to true. This information is not available in Bonita BPM Portal.
```

Comment:
does it means this false falsy information from 7.5 ?

Reply:
yes, it was